config/Migrations/20150511062806_initial.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class Initial extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-change-method
+     *
+     * Uncomment this method if you would like to use it.
+     *
+    public function change()
+    {
+    }
+    */
+    
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+		$sql = <<<SQL
+CREATE TABLE IF NOT EXISTS `queued_tasks` (
+  `id` int(11) NOT NULL,
+  `jobtype` varchar(45) NOT NULL,
+  `data` text,
+  `task_group` varchar(255) DEFAULT NULL,
+  `reference` varchar(255) DEFAULT NULL,
+  `created` datetime DEFAULT NULL,
+  `notbefore` datetime DEFAULT NULL,
+  `fetched` datetime DEFAULT NULL,
+  `completed` datetime DEFAULT NULL,
+  `progress` float DEFAULT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `failed` int(3) NOT NULL DEFAULT '0',
+  `failure_message` text,
+  `workerkey` varchar(45) DEFAULT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci AUTO_INCREMENT=1 ;
+ALTER TABLE `queued_tasks`
+  ADD PRIMARY KEY (`id`);
+SQL;
+		$this->query($sql);
+    
+    }
+
+    /**
+     * Migrate Down.
+     */
+    public function down()
+    {
+
+    }
+}
```

Comment:
Please make a new file with only the change you need, this first iniial one should not be touched